content/en/guides/diversity-inclusion.md

```diff
@@ -0,0 +1,173 @@
+
+This lower representation in technology overall points to a shortage of talent overall, and in turn, to a limitation in attracting new contributors into open source communities. Talent shortages slow innovation and up the ante between open source communities  competing for the available talent. It also hamstrings corporate contributors who are desperately seeking to add open source talent to their own rosters too.
+
+According to a Linux Foundation’s [Open Source Jobs Report](https://www.linuxfoundation.org/publications/2018/06/open-source-jobs-report-2018/), "87% of hiring managers report difficulty finding open source talent, and nearly half (48%) report their organizations have begun to support open source projects with code or other resources for the explicit reason of recruiting individuals with those software skills." 
+
+Open source communities come under risk if the pool of contributors and maintainers is not maintained at healthy levels which can lead to [open source burnout](https://opensource.com/article/19/11/burnout-open-source-communities).
+
+> "I'm concerned about the fact that a lot of our projects are going to be very under-resourced. Many projects that a lot of us care about have one or two developers only, and a lot of us are becoming older and gray haired," said [Nithya Ruff](https://www.linkedin.com/in/nithyaruff/), senior director, Open Source Practice at Comcast and a member of the Board of Directors at The Linux Foundation. “As we age, I'm concerned that, in our passing on the knowledge or expanding the contributor community and key projects such that we can sustain those projects, and the only way we can change that is if we open the doors and we do apprenticeships and mentorships and we actually pass the knowledge on and teach people and not just stay with, ‘Ah, it's me, and it'll die when I leave.’" But there’s more at stake than talent pipelines, succession plans, and the success of open source communities. “The other concern is as newer and newer industries get involved in additional transformation and start using open source, it's as much about scaling and welcoming new industries into open source,” says Ruff. “Open source needs to evolve and change and embrace new industries and new people into this and be able to codify and share the culture with them and not just stay static.”
```

Comment:
The leading ">" will cause this entire paragraph to be a blockquote, but only the first part is a quote.

Reply:
Nithya was already introduced on line 27. The LinkedIn link should be moved there and some of this duplication removed, or leave until here for the full introduction.